src/components/compass-schema/compass-schema.jsx

```diff
@@ -40,18 +37,14 @@ class Schema extends Component {
   static propTypes = {
     actions: PropTypes.object,
     store: PropTypes.object.isRequired,
-    samplingState: PropTypes.oneOf([
+    analysisState: PropTypes.oneOf([
       'initial',
-      'counting',
-      'sampling',
       'analyzing',
       'error',
       'complete',
-      'outdated',
       'timeout'
```

Comment:
How do you feel about having the `analysisState` be a constant? I think sometimes a lot of string literal comparisons in code can lead to accidental typos/bugs. Maybe we can address that at a different time though.

Reply:
That's a great idea